pdns/receiver.cc

```diff
@@ -590,8 +590,7 @@ int main(int argc, char **argv)
       }
     }
 
-    if(!::arg().mustDo("disable-tcp"))
-      TN=new TCPNameserver; 
+    TN = std::make_unique<TCPNameserver>();
```

Comment:
C++11 doesn't provide `make_unique` in the `std` namespace, we do have our own `make_unique` in `misc.hh`, though.